src/app/components/Hits/Hits.jsx

```diff
@@ -100,7 +127,10 @@ class Hits extends React.Component {
     return (
       <div id="results-description" className="nypl-results-summary">
         {
-          hits !== 0 ?
+          this.state.spinning ?
```

Comment:
Can you make these into separate variables so you don't have nested tertiary operations? ESlint will complain about that, and it's also easier to read.